enhancements/installer/packet-ipi.md

```diff
@@ -0,0 +1,197 @@
+
+## Design Details
+
+### Test Plan
+
+We will use existing UPI platforms, such as OpenStack, AWS, and GCP, as the
+inspiration for our testing strategy:
+
+- A new e2e job will be created
+- At the moment, we think unit tests will probably not be necessary. However, we
+  will cover any required changes to the existing codebase with appropriate
+  tests.
+
+### Graduation Criteria
+
+The proposal is to follow a graduation process based on the existence of a CI
+running suite with end-to-end jobs. We will evaluate its feedback along with
+feedback from QE's and testers.
+
+We consider the following as part of the necessary steps:
+
+- CI jobs present and regularly scheduled.
+- IPI document published in the OpenShift repo.
+- End to end jobs are stable and passing and evaluated with the same criteria as
+  comparable IPI providers.
+- Developers of the team have successfully used the IPI to deploy on Packet
+  following the documented procedure.
+
+## Implementation History
+
+Significant milestones in the life cycle of a proposal should be tracked in
+`Implementation History`.
+
+## Drawbacks
+
+The IPI implementation is provisioned on baremetal which faces resource
+availability limitations beyond those in VM environment. CI, QE, documentation,
+and tests will need to be generous when defining provisioning times. Tests and
+documentation should also be made flexible about facilities and device model
+choices to avoid physical limitations.
+
+## Alternatives
+
+People not using the IPI workflow can follow the [Packet
+UPI](https://github.com/RedHatSI/openshift-packet-deploy/blob/master/terraform/README.md)
+document. This requires more manual work and the necessary knowledge to identify
+Packet specific parts without any automation help.
+
+Users may also follow along with the [Deploying OpenShift 4.4 on
+Packet](https://www.openshift.com/blog/deploying-openshift-4.4-on-packet) blog
+post, but there is no automation provided.
+
+ProjectID is being defined as a high level variable. Arguably, OrganizationID
+could take this position, as this represents the billing account. OrganizationID
+is inferred by ProjectID in the current proposal. In a variation where
+Organization is required at install, ProjectID could become a required or
+optional (inherited) property for each cluster. Keep in mind, Projects are one
+way to share a private network between nodes.
+
+## Infrastructure Needed
+
+As has been demonstrated in the Packet UPI, users will need access to a Packet Project, API Key, and no less than 4 servers. One of these will be used for bootstrapping while the other 3 represent the control plane. The fourth node, used for bootstrapping can be removed once the cluster is installed.
```

Comment:
It should be possible to deploy only 3 servers.

https://docs.openshift.com/container-platform/4.5/installing/installing_bare_metal/installing-bare-metal.html#installation-three-node-cluster_installing-bare-metal